packages/client/components/NewMeetingActions.tsx

```diff
@@ -27,7 +27,11 @@ const ActionRow = styled('div')({
   padding: 24,
   paddingTop: 16,
   [narrowScreenMediaQuery]: {
-    flexDirection: 'column'
+      flexDirection: 'column',
```

Comment:
Can we stick buttons to the bottom of the screen on mobile?
Probably something like:

```
 margin-top: auto;
 padding-bottom: 24px;
}
```

Reply:
Thanks for the review, would work on that